variable/CMakeLists.txt

```diff
@@ -101,5 +106,6 @@ endif()
 add_subdirectory(test)
 
 if(DYNAMIC_LIB)
```

Comment:
Minor point, this dynamic_lib block is commonly duplicated. Maybe better a separate cmake function?

Reply:
Will look into this in follow-up, where I am adding more code here for exporting.